packages/realm-network-transport/src/types.ts

```diff
@@ -52,3 +52,202 @@ export interface NetworkTransport {
         handler: ResponseHandler,
     ): void;
 }
+
```

Comment:
All of these types are more or less copy-pasted from node_modules/typescript/lib/lib.dom.d.ts.